src/components/accordion/AccordionItem.jsx

```diff
@@ -53,10 +55,12 @@ const AccordionItem = ({
     dispatch,
     reduceMotion,
     onItemSelect,
+    isControlled,
   } = useContext(AccordionContext);
   const [isHovered, setIsHovered] = useState(false);
 
   const isCollapsed = !expanded.includes(id);
+  const isDisabled = !isCollapsed && isControlled;
```

Comment:
I found this does not work when we're in `allowMultiple` mode. I think this value can be controlled by accordion and passed as prop. Take a look at my other comment on `src/components/accordion/Accordion.jsx`

Reply:
We discussed with @katarzynatobis that we don't need it anymore. Component which controls Accordion also has to control `aria-disable` attributes

Reply:
I don't fully agree. 

Considering the rule:
> If the accordion panel associated with an accordion header is visible, and if the accordion does not permit the panel to be collapsed, the header button element has aria-disabled set to true.

and how our version of accordion is working we need to implement is as follows:
-  When `Accordion` is within **uncontrolled** mode, aria-disabled in `AccordionItem` should be always `false`. `AllowMultiple` flag has nothing to do here because it's different property from keeping at least one item always opened. We don't provide that property on purpose because in our implementation we decided that each item can be always collapsed by default.
- When `Accordion` is in **controlled** mode, developers can implement mechanism to keep at leas one element opened and we need to provide him a way to pass aria-disabled to the item they decide to keep in that state.

Let's discuss tomorrow

Reply:
Thats exactly same conclusion we had with Kasia. Let's meet tomorrow :)

Reply:
commit with disabled state is reverted now :)